region/maridia/inner-green.json

```diff
@@ -1211,6 +1211,79 @@
                     ]}
                   ],
                   "devNote": "Other weapons take too long to kill the enemy."
+                },
+                {
+                  "name": "Jump with Momentum",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    "canPlayInSand",
+                    "canTrickyJump",
```

Comment:
The strat would still need to kill the right Evir.  A quick super shot, a pair of tricky plasma hits, or plasma+other.